swagger.yaml

```diff
@@ -1689,6 +1713,38 @@ components:
         attributes:
           $ref: "#/components/schemas/Attributes"
 
+    UserFlextime:
+      type: object
+      description: User data combined with their flextime information
+      required:
+        - user
+        - flextime
+      properties:
+        user:
+          type: object
+          description: User information
+          required:
+            - id
+            - firstName
+            - lastName
+          properties:
+            id:
+              type: string
+              format: uuid
+              description: User's unique identifier
+            firstName:
+              type: string
+              description: User's first name
+            lastName:
+              type: string
+              description: User's last name
+            email:
+              type: string
+              format: email
+              description: User's email address
```

Comment:
Do we really need to include email here, again it seems like unnecessary information, we already have this in the UI from the users request that happens on login and I don't see what use it would be here.
As such I'd say remove this also